Formula/httpstat.rb

```diff
@@ -0,0 +1,16 @@
+class Httpstat < Formula
+  desc "Curl statistics made simple"
+  homepage "https://github.com/reorx/httpstat"
+  url "https://github.com/reorx/httpstat/archive/v1.1.2.tar.gz"
```

Comment:
Oh sorry, did you remove the old version or retag? Anyway, please ignore this comment.